### DISCUSSION TITLE
Circuit definition of terms:  Global versus external, connections versus intersections, etc.

### DISCUSSION THREAD

**Discussion by tzortea** [scikit-rf/scikit-rf#1058] [Q&A] [answered]
What is the difference between 'global' and 'external' s-parameters for a Circuit?  What is 'inner' versus 'outer' ports?  What is the difference between 'connections' and 'intersections' and 'nodes'?

**Comment by jhillairet** [comment, marked as answer]
Following the [Circuit doc](https://scikit-rf.readthedocs.io/en/latest/tutorials/Circuit.html), inner ports are all connection between elements in a Circuit, while _outer_ ports are the ports defined by the `Port` object. The _global_ S-parameters are the scattering parameters of both “inner” and “outer” ports, while _external_ are only the S-param for the "outer" ports.

![image](https://github.com/scikit-rf/scikit-rf/assets/4642848/605f5f13-00ec-45ae-b2d5-fe94bc267b40)

connections' and 'intersections' and 'nodes' are essentially the same. Actually, an intersection can be made of multiple connections (at least one). A node or a connection are a link between two elements.

**Reply by tzortea** [reply]
Thanks
